### PR TITLE
Test annotations

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

**Comment by MartinBasti** [comment]
I reused existing test, to test if package_name replacement correctly uses suffix.

I'm not sure if it's worth to create a separate namespace and separate test

File: tests/integration/alter_package_names/alter_package_names_test.py [hunk 1]
```diff
@@ -11,6 +11,24 @@
 from tests.integration.utils import test_env, make_bundle, is_yaml_file
 
 
+def _test_csv_annotations(quay_bundle, config):
```
Comment (csomh) [inline tests/integration/alter_package_names/alter_package_names_test.py]: How about calling this "check_csv_annotations"? As this is what it really does.

I would call something a "test" if has some object under test and pokes that object in order to check how the object reacts to that. Not sure if this makes sens :thinking:

File: tests/integration/alter_package_names/alter_package_names_test.py [hunk 2]
```diff
@@ -11,6 +11,24 @@
...
+    """
+    Test if bundle contains configured annotations.
+    Package name must contain suffix.
+    """
+
+    package_name = config["package"] + config["suffix"]
```
Comment (csomh) [inline tests/integration/alter_package_names/alter_package_names_test.py]: I would say that it's out of scope of this function to check that the package name is correct.

Just use `quay_bundle["packageName"]`.

If this is done, the function signature could be changed to:

```python
def check_csv_annotations(quay_bundle, expected_annotations):
```

Which maybe would be a little bit nicer.

File: tests/integration/alter_package_names/alter_package_names_test.py [hunk 3]
```diff
@@ -11,6 +11,24 @@
...
+
+    substitutions = {
+        'package_name': package_name,
+    }
+
+    for name, value in config['csv_annotations'].items():
```
Comment (csomh) [inline tests/integration/alter_package_names/alter_package_names_test.py]: Maybe double quotes for consistency?

**Comment by MartinBasti** [comment]
Updated, please check

**Pull request merged**